types/networks/src/lib.rs

```diff
@@ -15,41 +15,75 @@ mod drand;
     not(feature = "devnet"),
     not(feature = "mainnet"),
     not(feature = "conformance"),
+    not(feature = "calibnet")
 ))]
 compile_error!(
     "No network feature selected. Exactly one of \"mainnet\", \"devnet\", \"interopnet\", or \"conformance\" must be enabled for this crate."
 );
 
 #[cfg(all(
     feature = "mainnet",
-    any(feature = "interopnet", feature = "devnet", feature = "conformance",)
+    any(
+        feature = "interopnet",
+        feature = "devnet",
+        feature = "conformance",
+        feature = "calibnet"
```

Comment:
This feature is also not defined in the crate's `Cargo.toml`.

Reply:
:+1: